bedrock/products/templatetags/misc.py

```diff
@@ -495,9 +548,8 @@ def relay_bundle_savings(ctx, country_code=None, lang=None, product="relay-bundl
         {{ relay_bundle_savings(country_code=country_code, lang=LANG) }}
     """
 
-    available_plans = _relay_get_plans(country_code, lang, product)
-    selected_plan = available_plans.get(RELAY_12_MONTH_PLAN)
-    saving = float(selected_plan.get("saving"))
-    saving = int(saving * 100)
+    # there's only one supported region
+    savings = settings.RELAY_STRIPE_PLAN_DATA["bundle"]["prices"]["USD"]["saving"]
+    saving = int(savings)
     markup = f"{saving}"
     return Markup(markup)
```

Comment:
Similar to above, this looks like it can be simplified to `return Markup(savings)` since `savings` is already an integer in settings and `Markup` will do the string conversion for you.